### PR TITLE
Show kubernetes services

### PR DESCRIPTION
**Pull request by K-Phoen**
## Hey, I just made a Pull Request!

First part of #4064: adding support for Service manifests in the k8s plugin.

If it looks good to you, I'll add some tests and then open another PR for ingresses.

Screen Shot 2021-01-14 at 16 11 25
Screen Shot 2021-01-14 at 16 11 37
Screenshot from 2021-01-14 16-13-56
Screenshot from 2021-01-14 16-14-09

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by K-Phoen** [comment]
Rebased and added support for Service + Ingress manifests. Should be ready for review now :)

File: plugins/kubernetes/src/components/IngressesAccordions/IngressDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}: {
+  ingress: ExtensionsV1beta1Ingress;
+  expanded?: boolean;
+}) => {
+  return (
+    <KubernetesDrawer
+      object={ingress}
+      expanded={expanded}
+      kind="Ingress"
+      renderObject={(ingress: ExtensionsV1beta1Ingress) => {
```
Comment (Rugvip) [inline plugins/kubernetes/src/components/IngressesAccordions/IngressDrawer.tsx:34]: Just return `ingress.spec`? :grin:

Same below

**Review by Rugvip — APPROVED** [review]
Yup, thanks! 👍  :grin:

:shipit:

File: plugins/kubernetes/src/components/IngressesAccordions/IngressDrawer.tsx (hunk 1 above)
Comment (K-Phoen) [reply plugins/kubernetes/src/components/IngressesAccordions/IngressDrawer.tsx:34]: :facepalm: 

Updated, thanks!

**Pull request merged**